### DISCUSSION TITLE
Instrument Type

### DISCUSSION THREAD

**Discussion by JaminShanti** [jamulussoftware/jamulus#2746] [Ideas]
This probably won't create any waves, but I play ,
flute/clarinet/bassClarinet/SopranoAltoTenorBariSaxophone/Trumpet/Fhorn/Trombone
I played a long time to learn that many instruments.  
The problem I have with the instrument type, because I have to set it, no one ever asked me to switch.
I think Guitar/Bass players have the same thing going on.
Is there anyway we can add it as a hover option?  I know there isn't enough room to list multiple instruments on the main display.
This is some what of a "winmore" feature request, so maybe something for the backlog if there is interest.

**Comment by pljones** [comment]
You can change instrument by RPC, I think.

But don't you have to spend some time switching instrument, anyway, during which you're able to interact with Jamulus?  It seems a very obscure situation where that would _not_ be the case.

**Comment by dcorson-ticino-com** [comment]
Lots of people play other instruments than shown.  
This is no problem, talk with the others, suggest what is possible, put down one, pick up the next.
The instrument type is also just a suggestion, it is not a binding definition of what you have to play.

**Reply by JaminShanti** [reply]
What I usually do is default to listener before I decide. Then switch it if i can tell people are using the icon to make decisions.  This makes me somewhat of a lurker but is the best alternative. Btw people do use the icon for decisions and don't like it when the icon is wrong.

**Reply by pljones** [reply]
I agree - it's not great to use the _wrong_ icon.  "Listener" or "None" are fine.  Everyone _should_ listen before joining in on an open jam anyway - so they'll hear what you're playing.  It's better than saying you're playing bassoon but actually playing trumpet, I guess.  (Although there's generally little contention over, say, "bass-y" wind instruments and "lead-y" wind instruments, so if you pick one that represents what you're doing - even if it's not all of what you're doing, I doubt people would complain.  In jazz sessions, it's usually fairly easy to have multiple in each range.)

**Reply by pljones** [reply]
(That does almost argue for cutting down the number of instruments to "lead", "backing", "rhythm", "other" or something really simple...)

**Reply by JaminShanti** [reply]
Something to think about.  This is a win more feature.  I wouldn't touch it unless they're is more interest.  I'll be fine. No sleep lost.
